cargo build is throwing errors on mac (M1)

In Mac OS (M1), the project is throwing errors
```
error: cannot find macro `is_aarch64_feature_detected` in this scope
  --> src/trace/consolidation/fill_indices.rs:42:8
   |
42 |       if is_aarch64_feature_detected!("neon") {
   |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: a macro with a similar name exists: `is_x86_feature_detected`
   |
   = note: consider importing this macro:
           std::arch::is_aarch64_feature_detected

error[E0425]: cannot find value `fill_indices_aarch64_neon` in this scope
  --> src/trace/consolidation/fill_indices.rs:43:20
   |
43 |         selected = fill_indices_aarch64_neon;
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

warning: unused import: `crate::utils::next_multiple_of`
 --> src/trace/consolidation/fill_indices.rs:3:5
  |
3 | use crate::utils::next_multiple_of;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

For more information about this error, try `rustc --explain E0425`.
warning: `dbsp` (lib) generated 1 warning
error: could not compile `dbsp` due to 2 previous errors; 1 warning emitted
```

--- a/src/trace/consolidation/fill_indices.rs
+++ b/src/trace/consolidation/fill_indices.rs
@@ -39,7 +39,7 @@ fn select_impl(length: usize, indices: &mut Vec<usize>) {
     }
 
     #[cfg(target_arch = "aarch64")]
-    if is_aarch64_feature_detected!("neon") {
+    if std::arch::is_aarch64_feature_detected!("neon") {
         selected = fill_indices_aarch64_neon;
     }
 
@@ -265,7 +265,7 @@ pub unsafe fn fill_indices_aarch64_neon(original_length: usize, indices: &mut Ve
 
 /// Fills indices using wasm simd
 // FIXME: Add `target_family = "wasm"`/`target_arch = "wasm64"` support
-#[cfg(target_family = "wasm32")]
+#[cfg(target_arch = "wasm32")]
 #[target_feature(enable = "simd128")]
 unsafe fn fill_indices_wasm32_simd128(original_length: usize, indices: &mut Vec<usize>) {
     use std::arch::wasm32::{i32x4, i32x4_add, i32x4_splat, v128, v128_store};

